go.mod

```diff
@@ -251,6 +251,7 @@ require (
 	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
 	github.com/xi2/xz v0.0.0-20171230120015-48954b6210f8 // indirect
 	github.com/yashtewari/glob-intersection v0.2.0 // indirect
+	github.com/yosida95/uritemplate/v3 v3.0.2 // indirect
```

Comment:
It is *highly* suspicious that anyone would depend on this! It's unused and unmaintained.

Reply:
it is coming from the mcp dependency.